block/config.ml

```diff
@@ -1,6 +1,26 @@
 open Mirage
 
-let main = foreign "Unikernel.Main" (console @-> block @-> job)
+type shellconfig = ShellConfig
+let shellconfig = Type ShellConfig
+
+let config_shell = impl @@ object
+    inherit base_configurable
+
+    method configure i =
+      let dir = Info.root i in
+      Functoria_app.Cmd.run "dd if=/dev/zero of=%s/disk.img count=100000" dir
+
+    method clean i =
+      let dir = Info.root i in
+      Functoria_app.Cmd.run "rm -f %s/disk.img" dir
```

Comment:
@talex5 I agree.  could we discuss this in the functoria issue tracker, instead of here, please?